VehicleCode/Output/Dashboard.cpp

```diff
@@ -0,0 +1,90 @@
+#include "Dashboard.h"
+#include "Actuator.h"
+#include "Screen.h"
+#include "LEDArray.h"
+#include "Servo.h"
+#include "Horn.h"
+#include "../SensorID.h"
+
+// #include <Arduino.h>
+// #include <FlexCAN_T4.h>
+
+// Global variables
+#define BAUDRATE 250000
+
+// CHECK: define function
+// Constructor
+Dashboard::Dashboard(unsigned long startTime) {
+    // load parameters
+    this->startTime = startTime;
+
+    // Initialize CAN bus
+    FlexCAN_T4<CAN2, RX_SIZE_256, TX_SIZE_16> can2;
+    can2.begin();
+    can2.setBaudRate(BAUDRATE);
+
+    // Instantiate the actuators
+    Screen* lapScreen = new Screen();
+    Screen* totalScreen = new Screen(); 
+    LEDArray* batteryLife = new LEDArray(); 
+    LEDArray* batteryDraw = new LEDArray(); 
+    LEDArray* batteryTemp = new LEDArray(); 
+    Servo* speed = new Servo(); 
+    Horn* horn = new Horn(); 
+    
+    // Declare and initialize the display array of pointers
+    Actuator* display[] = {
+                            (Actuator*)lapScreen, (Actuator*)totalScreen, 
+                            (Actuator*)batteryLife, (Actuator*)batteryDraw, (Actuator*)batteryTemp, 
+                            (Actuator*)speed, 
+                            (Actuator*)horn
+                        };
+
+    numActuators = sizeof(display);
+}
+
+// CHECK: define function. idk if this is the most efficient approach
+/*!
+ * @brief Get the index of a sensor in the display array
+ * 
+ * @param sensor (SensorID) The sensor ID
+ * 
+ * @return (int) The index of the sensor in the display array
+ */
+int getSensorIndex(int id) {
+    SensorID sensorID = getSensorID(id);
```

Comment:
Maybe we just should be using enums, it seems like a lot of hoops to jump through